src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -445,23 +445,46 @@ export const GlobalProvider = memo(
         }, [hasRelevantUnsavedChanges, changeNodes, changeEdges, setSavePath, setViewport]);
 
         const performSave = useCallback(
-            (saveAs: boolean) => {
+            (saveAs: boolean, clearInputData = false, saveSeparately = false) => {
```

Comment:
I think these 2 parameters should be one `isTemplate: false`. `clearInputData` doesn't actually clear all input data, and `saveSeparately` doesn't really explain what it does.

Reply:
Good point